Add Spaces to the Using page (#452)
owncloud/docs-client-desktop: modules/ROOT/pages/using.adoc

image::using/account-force-sync-now.png[Force Sync Now, width=500,pdfwidth=60%]

=== Spaces

> Spaces are only available when connected to an Infinite Scale backend. In most cases they are similar to folders or external mounts when the backend is ownCloud Server and one might not see much differences though some exist:

Reviewer: ```suggestion
Spaces are only available when connected to an Infinite Scale backend. In most cases, they are similar to folders or external mounts when the backend is ownCloud Server and one might not see differences, though some exist:
```

> * On initial account setup, the available Spaces can be selected for the synchronisation process.

Reviewer 2: ```suggestion
* On initial account setup, all available Spaces are auto-selected for the synchronisation process.
```

> * On initial account setup, all available Spaces are auto-selected for the synchronisation process.

Reviewer: ```suggestion
* On initial account setup, all available spaces are auto-selected for the synchronization process.
```

> ** There are at minimum two Spaces added, which are the personal Space and the Space that contains all recieved shares. Shares are shown as folders in the local filesystem.

Reviewer: ```suggestion
** There are at minimum two spaces added, which are the personal space and the space that contains all received shares. Shares are shown as folders in the local filesystem.
```

* Post initial account setup, new spaces can be added via xref:using-an-account-that-connects-to-infinite-scale[Managing Resources to Synchronize].

> * When a space gets renamed in the WebUI, it also gets renamed in the sync window view but the local foldername where it is stored does not change.

Reviewer: ```suggestion
* When a space is renamed in the Web UI, it also gets renamed in the sync window view, but the local folder name where it is stored does not change.
```

> * Spaces can be customized with images in the WebUI. The image selected is also shown in the Desktop App.

Reviewer: ```suggestion
* Spaces can be customized with images in the Web UI. The image selected is also shown in the Desktop app.
```

> * When a Space gets disabled or deleted via the WebUI, no data is deleted locally.

Reviewer: ```suggestion
* When a space gets disabled or deleted via the Web UI, no data is deleted locally.
```

> * If you have used Spaces with the Desktop App version 3.0, you must remove the account from the Desktop App and do an initial account setup again.  

Reviewer 2: ```suggestion
* If you have used ownCloud Infinite Scale with the Desktop App older then 3.0, you must remove the account from the Desktop App and do an initial account setup again to work with Spaces.
```

> * If you have used ownCloud Infinite Scale with the Desktop App older then 3.0, you must remove the account from the Desktop App and do an initial account setup again to work with Spaces.

Reviewer: ```suggestion
* If you have used ownCloud Infinite Scale with a Desktop app version older then 3.0, you must remove the account from the Desktop app and do an initial account setup again to work with spaces.
```

=== Managing Resources to Synchronize

> When an account gets added the first time, you can select which remote resources you want to synchronize. These resources are represented by the Desktop app as local folders using the base path defined during account setup. Depending on the backend connected, resources can be folders and/or external mounts when using ownCloud Server or Spaces when it comes to Infinite Scale.

Reviewer: ```suggestion
When an account gets added the first time, you can select which remote resources you want to synchronize. These resources are represented by the Desktop app as local folders using the base path defined during account setup. Depending on the backend connected, resources can be folders and/or external mounts when using ownCloud Server or spaces when using Infinite Scale.
```

The base path defined on first sync is not the only path that can be used. Resources can have their individual base path. This is can be beneficial if you want selected resources to be located on different paths than the default.

==== Using an Account That Connects To ownCloud Server:

> * You will see a list of available remote resources you can manage as shown in the screenshot below. When a new resource gets added at the server like via the WebUI, it will be listed here.

Reviewer: ```suggestion
* You will see a list of available remote resources you can manage as shown in the screenshot below. If a new resource gets added on the server, e.g. via the Web UI, it will be listed here.
```

> * You will see a list of available remote resources you can manage as shown in the screenshot below. If a new resource gets added on the server, e.g. via the Web UI, it will be listed here.

Reviewer 2: only in non-VFS

Author: Hmmm, a resource is eg an external mount and if allowed but using vfs it is not listed?
How does one can say I would like this resource or do not like at all for syncing?

Author: And what about when the backend is ocis and you have with Spaces and you are usin vfs?

Reviewer 2: Let's have a meeting next week…

Author: fixed as discussed

> ** To add a resource to be synced as folder to the *same base path* or remove it from syncing, checkmark it to add or deselect to remove it. To finalize the selection, click the btn:[Apply] button on the bottom right.

Reviewer: ```suggestion
** To add a resource to be synced as folder to the *same base path*, set the check mark. To remove a resource from syncing, deselect it. To finalize the selection, click the btn:[Apply] button on the bottom right.
```

image::using/add-resource-server-same-folder.png[Add resource same path, width=300,pdfwidth=60%]

> ** To add a resource to be synced as folder to a *different path* than the account base path, click at the bottom left on btn:[Add Folder]. Select the path and the resources to sync and complete the process.

Reviewer: ```suggestion
** To add a resource to be synced as folder to a *different path* than the account base path, click in the bottom left corner on btn:[Add Folder]. Select the path and the resources to sync and complete the process.
```

image::using/add-resource-server-different-folder.png[Add resource different path, width=300,pdfwidth=60%]

==== Using an Account That Connects To Infinite Scale:

> When an account connects to Infinite Scale, the list of currently synced Spaces looks like the image below. It also shows the state of the synchronisation process:

Reviewer: ```suggestion
When an account connects to Infinite Scale, the list of currently synced spaces looks like the image below. It also shows the state of the synchronization process:
```

image::using/show-resources-ocis.png[Show resources Infinite Scale, width=300,pdfwidth=60%]

> When a new Space gets added via the WebUI, it will not be automatically listed here as this window only shows Spaces that are already synchronized.

Reviewer: ```suggestion
When a new space gets added via the Web UI, it will not be automatically listed here as this window only shows spaces that are already synchronized.
```

> To add a new Space for synchronisation, click the btn:[Add Space] button, select the Space and define the path if you do not want to take the default base location. Finalize with btn:[Add Sync Connection]:

Reviewer: ```suggestion
To add a new space for synchronization, click the btn:[Add Space] button, select the space and define the path if you do not want to take the default base location. Finalize with btn:[Add Sync Connection]:
```
